governance/remote_executor/programs/remote-executor/Cargo.toml

```diff
@@ -18,11 +18,13 @@ pythtest = []
 
 [dependencies]
 anchor-lang = {version = "0.25.0", features = ["init-if-needed"]}
-wormhole-solana = { git = "https://github.com/guibescos/wormhole", branch = "reisen/sdk-solana"}
-wormhole-core = { git = "https://github.com/guibescos/wormhole", branch = "reisen/sdk-solana"}
+wormhole-solana = { git = "https://github.com/guibescos/wormhole-solana", rev="f14b3b54c1e37e1aaf8c2ac2a5e236832ffdb3c2"}
```

Comment:
We should really get away from this, maybe we can chat about what needs to be upstreamed. We're running into this in pythnet too .

Reply:
This is a temporary home.